src/trix/config/block_attributes.coffee

```diff
@@ -5,6 +5,10 @@ Trix.config.blockAttributes = attributes =
   quote:
     tagName: "blockquote"
     nestable: true
+  heading:
```

Comment:
Also, I changed the name from *header* to *heading* since the HTML spec refers to `<h1>...<h6>` tags that way. "Headers" refers to [something else](http://www.w3schools.com/tags/tag_header.asp).